src/adapters/mod.rs

```diff
@@ -65,11 +79,8 @@ impl<T: Controller> AdapterManager<T> {
         ip_camera::IPCameraAdapter::init(manager, self.controller.clone()).unwrap();
         let scripts_path = &self.controller.get_profile().path_for("thinkerbell_scripts.sqlite");
         ThinkerbellAdapter::init(manager, scripts_path).unwrap(); // FIXME: no unwrap!
-        let profile_openzwave = &self.controller.get_profile().path_for("openzwave");
-
-        let openzwave_devices = self.controller.clone().get_config().get("openzwave", "devices");
-        OpenzwaveAdapter::init(manager, profile_openzwave, openzwave_devices).unwrap();
```

Comment:
you could do:
#[cfg(feature = "zwave")]
{
  let profile_openzwave = ...
}

I'm always unhappy with the empty methods when the feature is disabled.